zynqmp.mk

```diff
@@ -30,21 +38,27 @@ ifneq ($(PETALINUX_VER),2018.2)
 	$(error This makefile only support Petalinux 2018.2)
 endif
 
-petalinux-create: check-petalinux
-	@cd $(ROOT) && petalinux-create -t project -s $(BSP_PATH)
-	@cd $(PRJ_PATH) && petalinux-create -t apps --template install -n optee-client --enable
-	@cd $(PRJ_PATH) && petalinux-create -t apps --template install -n optee-test --enable
-	@#
+petalinux-create: check-petalinux	
+	@cd $(ROOT) && petalinux-create -n $(PLATFORM)-$(PETALINUX_VER) -t project -s $(BSP_PATH)
 	$(call set_cfg,CONFIG_SUBSYSTEM_ATF_COMPILE_EXTRA_SETTINGS,"SPD=opteed ZYNQMP_BL32_MEM_BASE=0x60000000 ZYNQMP_BL32_MEM_SIZE=0x80000",$(PRJ_PATH)/project-spec/configs/config)
 	$(call set_cfg,CONFIG_SUBSYSTEM_ZYNQMP_ATF_MEM_SIZE,0x16001,$(PRJ_PATH)/project-spec/configs/config)
 	@#
+	@# Replace BSP default rootfs by a minimal one to speed up building process and ease compatibility between different boards
```

Comment:
I know there are plenty of lines here exceeding 80 characters, but please try to fit things into 80 character if possible (at least for comments it should be easy to achive).

Reply:
Is it ok for you? Result is a little less readable, in my opinion.